config/settings.yml

```diff
@@ -579,50 +579,6 @@ hca:
     region: us-gov-west-1
   timeout: 30
 hostname: <%= ENV['hostname'] %>
-hqva_mobile:
-  development_key_path: modules/health_quest/config/rsa/sandbox_rsa
-  facilities:
-    ids_path: "/facilities_api/v1/va"
-    url: <%= ENV['hqva_mobile__facilities__url'] %>
-  lighthouse:
-    claim_common_id: <%= ENV['hqva_mobile__lighthouse__claim_common_id'] %>
-    client_assertion_type: urn:ietf:params:oauth:client-assertion-type:jwt-bearer
-    condition_aud_claim_url: https://deptva-eval.okta.com/oauth2/aus8nm1q0f7VQ0a482p7/v1/token
-    grant_type: client_credentials
-    health_api: health_api
-    health_api_path: "/services/fhir/v0/r4"
-    health_api_scopes:
-    - launch launch/patient
-    - patient/Appointment.read
-    - patient/Location.read
-    - patient/Organization.read
-    - patient/Patient.read
-    - patient/Condition.read
-    - patient/Observation.read
-    - patient/Practitioner.read
-    - patient/Medication.read
-    - system/Endpoint.read
-    health_aud_claim_url: <%= ENV['hqva_mobile__lighthouse__health_aud_claim_url'] %>
-    health_token_path: "/oauth2/health/system/v1/token"
-    host: <%= ENV['hqva_mobile__lighthouse__host'] %>
-    key_path: <%= ENV['hqva_mobile__lighthouse__key_path'] %>
-    mock: <%= ENV['hqva_mobile__lighthouse__mock'] %>
-    pgd_api: pgd_api
-    pgd_api_scopes:
-    - launch launch/patient
-    - patient/Observation.read
-    - patient/Observation.write
-    - patient/Questionnaire.read
-    - patient/Questionnaire.write
-    - patient/QuestionnaireResponse.read
-    - patient/QuestionnaireResponse.write
-    pgd_aud_claim_url: <%= ENV['hqva_mobile__lighthouse__pgd_aud_claim_url'] %>
-    pgd_path: "/services/pgd/v0/r4"
-    pgd_token_path: "/oauth2/pgd/v1/token"
-    redis_session_prefix: healthquest_lighthouse
-    url: <%= ENV['hqva_mobile__lighthouse__url'] %>
-  mock: <%= ENV['hqva_mobile__mock'] %>
-  url: <%= ENV['hqva_mobile__url'] %>
```

Comment:
Do you have a task for yourself to remove these from Parameter store post-deploy?

Reply:
> remove AWS values connected to deprecated hqva_mobile configurations

yep! 😄 I see this in the description now